internal/manager/config.go

```diff
@@ -234,16 +237,28 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	return flagSet
 }
 
-func (c *Config) GetKongClient(ctx context.Context) (*kong.Client, error) {
-	if c.KongAdminToken != "" {
-		c.KongAdminAPIConfig.Headers = append(c.KongAdminAPIConfig.Headers, "kong-admin-token:"+c.KongAdminToken)
-	}
-	httpclient, err := adminapi.MakeHTTPClient(&c.KongAdminAPIConfig)
-	if err != nil {
-		return nil, err
-	}
+func (c *Config) GetKongClients(ctx context.Context) ([]*kong.Client, error) {
+	var (
+		urls    = strings.Split(c.KongAdminURL, ",")
```

Comment:
nit:

I wonder if the interface wouldn't be more testable if this function accepted a slice of strings (URLs) and returned a corresponding slice of Kong clients.

Reply:
This actually got me thinking that this might need a bit more rewriting. Thanks! I'll post a revised version later on.

Reply:
Refactored this bit.